integration/skydb.test.js

```diff
@@ -0,0 +1,60 @@
+const { client, portal } = require(".");
```

Comment:
Moved into new `integration` folder and removed unnecessary `describe` blocks. I thought those were causing tests to run in parallel but it didn't make a difference - it is, however a bit cleaner without them. There were also some `it` tests with duplicate names which I renamed.

Reply:
hello m-cat,

I myself have done a test with several portals and then, if it was feasible, used different portals, so the tests ran quickly.
But I'm not sure if you mean these.